### ISSUE TITLE
Rename option - Should we call it something else?

### ISSUE THREAD

**Issue by scruffian** [WordPress/gutenberg#56240]
## What problem does this address?
Some users expect the "Rename" option to actually change the content of the the block, for example a heading or a navigation item.

## What is your proposed solution?
Should we consider a different label for the "Rename" feature to make it clearer that this is just a label for the block? Here are some ideas...

- Rename block
- Label block
- Change label
- Change name
- Customize label
- Customize name

Are there better options?

**Comment by richtabor** [comment]
I don’t think so. I don’t think any of those options would reduce that behavior. If anything, maybe the modal can be more verbose at explaining what’s happening perhaps.

**Comment by Mamaduka** [comment]
Is this still relevant? The feature has been out for a couple of WP releases now, and I've not seen any reports about it being unclear.
